app/com/kinja/play/plugins/Closure.scala

```diff
@@ -45,7 +43,7 @@ class InvalidClosureValueException(obj: Any, path: Option[String] = None) extend
 /**
  * Play plugin for Closure.
  */
-class ClosurePlugin(app: Application) extends Plugin {
+class ClosurePlugin @Inject() (implicit app: Application) extends Plugin {
```

Comment:
Right, I was just wondering if we would have to come back and change this later.